content/applications/finance/accounting/bank/feeds/bank_synchronization.rst

```diff
@@ -38,49 +33,49 @@ following address:
 
 - https://production.odoofin.com/
 
-First Synchronization
+First synchronization
 ---------------------
 
-You can start synchronization either by going to :menuselection:`Accounting --> Configuration
---> Add a Bank Account` or via the configuration bar on the accounting dashboard.
+You can start synchronization either by going to :menuselection:`Configuration
+--> Add a Bank Account` or via the bank journal's configuration on the dashboard.
 
 Now you can search for your bank institution. Select it and follow the steps to synchronize with it.
 
 .. note::
    If you have any issues during your first synchronization, please verify that your
-   web browser doesn't block pop-ups and that your adblocker is disabled.
+   web browser doesn't block pop-ups and that your ad-blocker is disabled.
+
+.. important::
+   When choosing the date for the first bank statements synchronization, pick the actual date where
+   you start working on your Odoo accounting database. For example, if you import your closing
+   balance in Odoo on the 31.12.2022 and you start recording accounting transactions on the
+   01.01.2023, your synchronization date should be 01.01.2023.
```

Comment:
I'd rephrase this a bit:
```suggestion
.. important::
   When choosing the date for the first bank statement synchronization, pick the date of when you
   start working on your Odoo accounting database. For example, if you import your closing balance 
   in Odoo on 31.12.2022, but start recording accounting transactions on 01.01.2023, your
   synchronization date should be 01.01.2023.

```
The most important parts in this suggestion are:
- no "the" before the dates
- ",but start" -> I think it makes more sense that way